Bulldozer.CSV/CSVComponent.cs

```diff
@@ -194,244 +397,388 @@ public override bool LoadSchema( string fileName )
         /// <returns></returns>
         public override int TransformData( Dictionary<string, string> settings )
         {
-            var importUser = settings["ImportUser"];
+            SqlServerTypes.Utilities.LoadNativeAssemblies( AppDomain.CurrentDomain.BaseDirectory );
+
+            ReportProgress( 0, "Staging csv data..." );
+
+            // only import things that the user checked
+            var selectedCsvData = CsvDataToImport.Where( c => c.TableNodes.Any( n => n.Checked != false ) ).ToList();
```

Comment:
For discussion next time. We moved this up from after the LoadExistingData checks, but should it be moved up? Or should we still do the health checks first before loading the csv files? Is something needed in the health checks from the CSV?